src-built-in/components/apitest/test1.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+		<meta name="description" content="" />
+		<meta name="viewport" content="width=device-width, initial-scale=1" />
+		<link rel="stylesheet" type="text/css" href="test1.css" />
+		<script src="test1.js"></script>
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+		<link
+			href="https://fonts.googleapis.com/css?family=Open+Sans+Condensed"
+			rel="stylesheet"
+		/>
+		<script src="../../vendor.bundle.js"></script>
+	</head>
+
+	<body>
+		<label for="symbolInput" id="symbolLabel">Input text</label>
```

Comment:
What is input text used for, it isn't clear.

Reply:
My intention is to allow the tester to input some data to test some data related apis. E.g. pub sub...

Reply:
I discovered some of this when going through the tests. You may want to put a comment on the form or a placeholder in the text field.